Main.qml

```diff
@@ -2,33 +2,149 @@ import QtQuick
 import QtQuick.Controls
 import com.myobject
 
+import QtQuick.Layouts
+import QtQuick.Controls.Universal
+
 Window {
     width: 640
-    height: 480
+    height: 750
     visible: true
-    title: qsTr("Hello World")
+    title: qsTr("LivePose_test")
 
     DataModel {
         id: x
         Component.onCompleted:
         {
-            console.log(data);
-            data.foo = "hiiii";
-            console.log(data);
-            console.log(JSON.stringify(data));
-        }
+            // Set pose_backend as backend
+            x.pose_backend.name = "PoseNet";
+            x.posenet_params.base_model = "resnet50";
+            x.posenet_params.device_id = 0;
+            x.posenet_params.draw_bounding_boxes = false;
+            x.posenet_params.draw_skeleton_edges = true;
+            x.posenet_params.input_resolution = [250, 250];
+            x.posenet_params.min_keypoint_confidence = 0.1;
+            x.posenet_params.min_pose_confidence = 0.15;
+            x.posenet_params.multi_pose_detection = true;
+            x.posenet_params.output_stride = 32;
+            x.pose_backend.params = x.posenet_params;
+            x.config.pose_backends = [ x.pose_backend ];
+
+            // Set up cameras
+            x.camera_params.resolution = [640, 480];
+            x.camera_params.framerate = 30;
+            x.camera_params.flip = false;
+            x.camera_params.rotate = 0;
+            x.cameras.input_paths = ["/dev/video0"];
+            x.cameras.params["/dev/video0"] = x.camera_params;
+            x.config.cameras = [ x.cameras ];
+
```

Comment:
x.cameras.params["/dev/video0"] = x.camera_params;
            
yes for this we need a different approach. x.cameras.params must be a QVariantMap as it will allow to use arbitrary "keys" - right now keys have to be a C++ variable, and a C++ variable cannot be named /dev/video0 of course.